ApplicationLibCode/ProjectDataModel/Summary/RimSummaryCurveAutoName.cpp

```diff
@@ -43,26 +45,27 @@ RimSummaryCurveAutoName::RimSummaryCurveAutoName()
     // clang-format off
```

Comment:
Remove clang-format? Should at least have leading and trailing space next to parens if not.